src/components/Button.js

```diff
@@ -0,0 +1,17 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const Button = ({ name }) => (
+  <div>
+    {name}
+  </div>
+);
+Button.propTypes = {
+  name: PropTypes.string,
+};
+
+Button.defaultProps = {
+  name: 'Button',
+};
```

Comment:
is this necessary ??? 
if it is, it should be an empty string `' '`

Reply:
No, my mistake, I read again the requirements and it's no necessary to set default props in this component, thanks!